server/game/cards/09.4-TCoH/RighteousDelegate.js

```diff
@@ -0,0 +1,40 @@
+import { Durations } from '../../Constants.js';
+
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class RighteousDelegate extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'increase the skill of each non-bushi and decrease the skill of each bushi',
```

Comment:
This is probably too long of a title (and it should be capitalised at the start)